rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -602,19 +593,6 @@ def _create_features(
 
         return sequence_features, sentence_features
 
-    def _get_featurized_attribute(
-        self, attribute: Text, all_tokens: List[List[Text]]
-    ) -> Tuple[
-        List[Optional[scipy.sparse.spmatrix]], List[Optional[scipy.sparse.spmatrix]]
-    ]:
-        """Return features of a particular attribute for complete data"""
-
-        if self._check_attribute_vocabulary(attribute):
```

Comment:
why can we omit this check?

Reply:
Made sense in the moment, will check again

Reply:
If you look into the code that calls this, you'll see that this is only called for an already trained model. The model is explicitly trained for the given attributes. I think it is really wrong here to just return an empty list for something that the user intends to be there. I think that's error obfuscation. What do you think?

Reply:
but seems there is still a failing test with this change ... gotta investigate :)

Reply:
Well good that we have these tests, I was wrong, the attributes are there in a very general fashion and if there's no data for them you just get empty vectorizers. Found a much simpler change too :)